lib/manager/cake/__fixtures__/build.cake

```diff
@@ -1,5 +1,6 @@
 foo
 #addin nuget:?package=Foo.Foo&version=1.1.1
+#addin "nuget:?package=Bim.Bim&version=6.6.6"
```

Comment:
What about other quote styles? E.g. `"""foobar"""`, `'''foobar'''` or ``  `foobar` ``

Reply:
The are not documented and I've never ever seen anything other than `"` out in the wilds. Will run some quick checks to be sure.